Jenkinsfile

```diff
@@ -21,9 +21,9 @@ pipeline {
                     make -f docker.Makefile test-e2e-non-experimental"
             }
         }
-        stage("e2e (non-experimental) - 18.09 engine") {
+        stage("e2e (non-experimental) - 19.03 engine") {
```

Comment:
question: can't we bump to 20.10 here too ?

Reply:
This one is specifically to run agains an older version of the engine (others should be running on latest)